src/oscar/apps/order/abstract_models.py

```diff
@@ -223,24 +223,23 @@ def shipping_tax(self):
 
     @property
     def shipping_status(self):
-        events = self.shipping_events.all()
+        events = self.shipping_events.order_by('-date_created').all()
```

Comment:
We should order by PK here as well, in case two timestamps are identical. Do we have a guarantee that PKs are created in order?